content/changelog/2024-04-11-clever-tools-3.5.2.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Clever Tools 3.5.2 is available with cURL, KV, Tasks and more!
+date: 2024-04-11
+tags:
+  - clever-tools
+  - cli
+authors:
+  - name: David Legrand
+    link: https://github.com/davlgd
+    image: https://github.com/davlgd.png?size=40
+  - name: Hubert Sablonière
+    link: https://github.com/hsablonniere
+    image: https://github.com/hsablonniere.png?size=40
+description: Lots of new features included!
+excludeSearch: true
+---
+
+Clever Tools 3.5 is now available, and this branch brings lots of new important features. First, `clever curl` is now public and listed as an official command. It helps you to send cURL requests with the auth context of your Clever Tools configuration. Thus, you can use Clever Cloud [API v2 or v4](/api/)..
+
+JSON format is supported for more commands, the `--since` option now supports a duration value. For example if you want to get logs since 2 hours ago, you can use `clever logs --since 2h`. This command documentation is available [there](https://github.com/CleverCloud/clever-tools/blob/master/docs/applications-deployment-lifecycle.md#logs).
+
+Clever Tasks can now be easily created and configured from Clever Tools. They are applications which can be run on demand, not awaiting any HTTP request on the `8080` port, but needing a `CC_RUN_COMMAND` to execute. Once it's done, the application stops. You're just billed for the execution time. It can help you to make some checks, compilation, file conversions, etc.
```

Comment:
```suggestion
Clever Tasks can now be directly created and configured from Clever Tools. They're applications which can be run on demand, not awaiting any HTTP request on the `8080` port, but needing a `CC_RUN_COMMAND` to execute. Once it's done, the application stops. You're just billed for the execution time. It can help you to make some checks, compilation, file conversions, etc.
```